src/NativeShell/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "org.jellyfin.cordova",
+  "version": "1.0.0",
+  "main": "nativeshell.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "GPL-2.0-only",
```

Comment:
Hmm, if this is new code maybe license it as GPLv2+? So we can move to GPLv3 eventually